docs/data/routes/docs/fundamentals/services/dictionary/overview/en.md

```diff
@@ -0,0 +1,26 @@
+---
+name: overview
+routeTemplate: ./data/component-templates/article.yml
+title: Dictionary Services and APIs
+---
+
+# Dictionary Services and APIs
+
+Often multilingual apps will need a dictionary of static phrases that require translation. Commonly this would be items such as form labels, global navigation items, footers, etc. 
+
+## Sitecore endpoints for dictionary data
+
+You have several options for fetching dictionaries from Sitecore: 
+
+- [Sitecore Dictionary Service](/docs/fundamentals/services/dictionary/sitecore-dictionary-service),
+- [Sitecore Experience Edge for XM](#sitecore-experience-edge-for-xm) or 
+- [Sitecore GraphQL](#sitecore-graphql).
+
+### Dictionary Service
+The [Sitecore Dictionary Service](/docs/fundamentals/services/dictionary/sitecore-dictionary-service) is a REST endpoint providing app-specific dictionary data.
+### Sitecore Experience Edge for XM
+[Sitecore Experience Edge for XM](https://doc.sitecore.com/developers/101/developer-tools/en/introducing-sitecore-experience-edge-for-xm.html) comes with a read-only GraphQL schema that is designed to accommodate common front-end use cases for headless Sitecore development. Read about the [Experience Edge schema](https://doc.sitecore.com/developers/101/developer-tools/en/the-experience-edge-schema.html).
+### Sitecore GraphQL
+[Sitecore GraphQL API](/docs/fundamentals/services/graphql) is an implementation of a GraphQL server on top of Sitecore. The Sitecore GraphQL schema mirrors the [Experience Edge schema](https://doc.sitecore.com/developers/101/developer-tools/en/the-experience-edge-schema.html).
+## JSS Dictionary API
+The [JSS Dictionary API](/docs/fundamentals/services/dictionary/jss-dictionary-api) is a suite of services, clients and data fetchers for either REST or GraphQL.
```

Comment:
I don't think it's clear how this sections is related to the content above it. I think we should be more explicit about the "why"... something like "the front-end app does not need to care about which method from above you are using, as the data fetching details are abstracted behind sitecore-jss's i18n API"

This is the whole point, the value that JSS brings